Lib/selectors.py

```diff
@@ -470,9 +488,11 @@ def select(self, timeout=None):
                 return ready
             for fd, event in fd_event_list:
                 events = 0
-                if event & ~select.EPOLLIN:
+                if event & ~(self._EVENT_READ | self._EVENT_WRITE):
```

Comment:
Can we precompute the masks to move any unnecessary computation out of the loop?